.github/workflows/tests.yaml

```diff
@@ -30,3 +30,6 @@ jobs:
         pip install tox tox-gh-actions
     - name: Test with tox
       run: tox
+    - uses: olafurpg/setup-scala@v10
+    - name: Check scienceworld compilation
+      run: ./simulator/package.sh
```

Comment:
Ideally, we want to use that compiled jar file to run the tests as well.

Reply:
Artifact can be share between runs https://docs.github.com/en/actions/using-workflows/storing-workflow-data-as-artifacts